File: 0009-palindrome-number/0009-palindrome-number.js
/**
 * @param {number} x
 * @return {boolean}
 */
var isPalindrome = function(x) {
    x_str = x.toString();
    x_str_rev=x_str.split('').reverse().join('');
    if(x_str == x_str_rev){
        return true;
    }
    else {
        return false;
    }
    
};

File: 0009-palindrome-number/0009-palindrome-number.py
class Solution(object):
    def isPalindrome(self, x):
        """
        :type x: int
        :rtype: bool
        """
        x_str = str(x)
        x_str_rev = x_str[::-1]
        if(x_str == x_str_rev):
            return True
        else:
            return False